DOCSP-16494 consistent purpose statements (#103)
mongodb/docs-java: source/fundamentals/data-formats/documents.txt

Overview
--------

> In this guide, you can learn how you can work with **documents** in the

Reviewer: ```suggestion
In this guide, you can learn how to use **documents** in the
```

Suggestion: similar to previous suggestion, "you can learn how you can" I think should be replaced.
